src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
-export interface Figure {
-
+enum Shape {
+  triangle = 'triangle',
```

Comment:
By convention, we capitalize enum keys
```suggestion
  Triangle = 'triangle',
```

Reply:
remembered and changed.